draft-ietf-quic-qpack.md

```diff
@@ -311,7 +311,7 @@ instructions received on the encoder stream.
 The decoder MUST emit header fields in the order their representations appear in
 the input header block.
 
-### Blocked Decoding
+### Blocked Decoding  {#blocked-decoding}
```

Comment:
I think auto anchoring should mean this is not required (I.e. only need to define explicit links that differ from the section title)

Reply:
A) did not know about auto-anchoring
B) seems like it didn't work for this specific one, since draft-10 of the qpack document has an unresolved reference to the section (see https://tools.ietf.org/html/draft-ietf-quic-qpack-10#section-2.1.3). Not sure why that's failing then...

Reply:
The reference in 2.1.3 should have `{{` instead of `{`, looks like.

Reply:
Aha, that might do the trick. Did that instead in 479c7d0

Reply:
Just built it locally to confirm; it works now.